app/searches/locations_search.rb

```diff
@@ -34,6 +34,53 @@ def search
     search_results
   end
 
+  def exact_match_found?
+    keyword_to_use = keywords || attributes[:keyword]
+
+    if keyword_to_use.blank? && matched_category.blank?
```

Comment:
Can you please add a comment on when this could happen?

Reply:
Sure, it would happen on an empty search or after a user clears the filters